homework_classes/src/homework_classes/Rectangle.java

```diff
@@ -0,0 +1,70 @@
+package homework_classes;
+
+/**
+ * Represents a geometry figure - rectangle. Contains method to calculate area
+ * and perimeter of the rectangle, based on rectangle sides
+ * 
+ * @author ajla
+ *
+ */
+public class Rectangle {
+	private double shorterSide;
+	private double largerSide;
+
+	/**
+	 * Constructor checking if entered values are larger then zero.
+	 * @param shorterSide
+	 * @param largerSide
+	 */
+	public Rectangle(double a, double b) {
+		try {
+			if (a > 0 && b > 0 && a != b) {
```

Comment:
Already mentioned, feel free to let illegal arg ex escape the method in case arguments are not valid.